nextflow_schema.json

```diff
@@ -797,7 +791,7 @@
                     "type": "boolean",
                     "fa_icon": "fas fa-download",
                     "description": "Download annotation cache.",
-                    "help_text": "Set this parameter, if you wish to download annotation cache."
+                    "help_text": "Set this parameter, if you wish to download annotation cache.\nUsing this parameter will download cache even if --snpeff_cache and --vep_cache are provided."
```

Comment:
I don't understand that. In what scenario would the user want to specify `--download_cache` along with `--snpeff_cache` and `--vep_cache`?  Doesn't it just indicate that the user doesn't know what he is doing?

Reply:
These params are specified by default, so it's most likely that a user will not notice that and try to download cache which will fail